Gemfile

```diff
@@ -96,7 +96,9 @@ gem "popper_js"
 gem "rsolr", ">= 1.0", "< 3"
 gem "rubytree"
 gem "rubyzip", ">= 1.3.0"
-gem "sidekiq", "~> 6.5"
+source "https://gems.contribsys.com/" do
+  gem "sidekiq-pro"
+end
```

Comment:
Issue: While it will probably work this way, it would be better if you could move that block (lines 99-101) to [somewhere around here](https://github.com/pulibrary/pulfalight/blob/main/Gemfile#L81) with a space above and below.